widget/Toaster.js

```diff
@@ -76,10 +78,11 @@ define([
 			this.hide();
 
 			// place node as a child of body for positioning
-			this.ownerDocument.body.appendChild(this.domNode);
+			this.domNode.ownerDocument.body.appendChild(this.domNode);
```

Comment:
Actually _WidgetBase has an `ownerDocument` property so you don't need this change.